libevmasm/Assembly.cpp

```diff
@@ -629,15 +629,18 @@ LinkerObject const& Assembly::assemble() const
 			ret.bytecode.resize(ret.bytecode.size() + bytesPerDataRef);
 			break;
 		case PushSub:
-			assertThrow(i.data() <= numeric_limits<size_t>::max(), AssemblyException, "");
```

Comment:
Why is this removed?

Reply:
I added it back, but since I am fixing few bugs still didn’t push